### DISCUSSION TITLE
Caminho Secreto

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#327] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/caminho-secreto/

**Comment by one-eyed-willy** [comment]
Nem me fala! Todo RPG que eu jogo eu fico nessa "mas vai que tem alguma coisa" e fico horas perdido no meio de um monte de inimigo muito mais forte que eu até voltar pras cidades principais
